Cargo.toml

```diff
@@ -18,6 +18,7 @@ actix-multipart = "0.6.0"
 actix-cors = "0.6.4"
 actix-ws = "0.2.5"
 actix-files = "0.6.2"
+actix-http = "3.4.0"
```

Comment:
It looks like this is only used in the tests, so we should put it into the [\[dev-dependencies\]](https://doc.rust-lang.org/rust-by-example/testing/dev_dependencies.html) section to keep it out of the main binary.